package.json

```
@@ -6,13 +6,18 @@
     "packages/*"
   ],
   "scripts": {
+    "build:libs": "npx nx run-many --target=build --projects=\"@root/**\"",
```

Right now we only have one library so the right command should be:
```suggestion
    "build:libs": "npx nx run build --project=@root/client",
```